graylog2-web-interface/src/views/components/widgets/SearchQueryExecutionInfoHelper.tsx

```diff
@@ -109,16 +115,21 @@ const SearchQueryExecutionInfoHelper = ({ currentWidgetMapping, children }: Prop
     setOpen((cur) => !cur);
   }, []);
 
-  return (
+  return interactive ? (
     <Popover position="bottom" opened={open} onClose={onClose} closeOnClickOutside>
       <Popover.Target>
-        <TargetContainer role="presentation" onClick={onToggle}>{children}</TargetContainer>
+        <TargetContainer role="presentation" onClick={onToggle}>
+          <>
+            {children}
+            <StyledIcon name="help" />
+          </>
+        </TargetContainer>
       </Popover.Target>
       <Popover.Dropdown title="Execution Info">
         {isEmpty(result) ? <i>No query executed yet.</i> : <HelpPopover widgetExecutionData={widgetExecutionData} />}
       </Popover.Dropdown>
     </Popover>
-  );
+  ) : <span>{children}</span>;
```

Comment:
I think we can just return `children` or `<>{children}</>` here and get rid of the `span`.